tools/docker/substrate-all-in-one/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+# From https://github.com/substrate-developer-hub/substrate-node-template
+version: "3.2"
+
+services:
+  dev:
+    container_name: contracts-node-template-cactus
+    image: hyperledger/substrate-all-in-one:latest
```

Comment:
@RafaelAPB @CatarinaPedreira Please update the image to the latest official that is published